src/Microsoft.DotNet.Build.Tasks.Packaging/src/GenerateNuSpec.cs

```diff
@@ -284,6 +287,13 @@ select reference.File
                     ).ToList();
         }
 
+        private List<PackageType> GetPackageTypes()
+        {
+            var packageTypes = from p in PackageTypes.NullAsEmpty()
+                               select new PackageType(p.ItemSpec, PackageType.EmptyVersion);
```

Comment:
We shouldn't need to set the PackageType version for anything right now (and it would probably be better to move to the real pack task, if that support was needed, anyways)